k8/deploy/genai-deploy.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: genai
+  namespace: team-promptheus
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: genai
+  template:
+    metadata:
+      labels:
+        app: genai
+    spec:
+      containers:
```

Comment:
I am fine with this for the demo deployment, but you reaaaly want CPU resource requests on contaieners.
I don't think that for our usecase we want limts.

Currently, this is done in the free floating way, k8s could schedule 10000 of this pod on the same machine.